sphinx/util/compat.py

```diff
@@ -10,31 +10,7 @@
 """
 from __future__ import absolute_import
 
-import warnings
-
-from docutils import nodes
 from docutils.parsers.rst import Directive  # noqa
 
 from docutils import __version__ as _du_version
 docutils_version = tuple(int(x) for x in _du_version.split('.')[:2])
-
-
-def make_admonition(node_class, name, arguments, options, content, lineno,
```

Comment:
Ah, that's very bothersome...
I will split this PR into some other PRs to merge this without `make_admonition`.

BTW, what do you think to remove descriptions of confvals.
For example, `html_translator_class` will be removed after merging. This entry also should be removed?
https://github.com/sphinx-doc/sphinx/blame/master/doc/config.rst#L949